deploy/olm-catalog/portworx/portworxoperator.v1.1.0.clusterserviceversion.yaml

```diff
@@ -69,71 +69,88 @@ spec:
   - name: Product Features
     url: https://portworx.com/products/features
   - name: Documentation
-    url: https://docs.portworx.com
+    url: https://docs.portworx.com/portworx-install-with-kubernetes/on-premise/openshift/operator
```

Comment:
the OLM framework appears to be generic and not tied to just openshift. Would it be okay to just link to openshift docs in that case?

Reply:
The specs from this directory only get published to openshift certified repo. For the generic OLM portworx operator we publish from here -https://github.com/operator-framework/community-operators/tree/master/upstream-community-operators/portworx
with slight modifications.
Will make it a point to not update this link in the community operators repo PR.